great_expectations/core/expectation_validation_result.py

```diff
@@ -613,7 +610,7 @@ def get_failed_validation_results(
     ) -> ExpectationSuiteValidationResult:
         validation_results = [result for result in self.results if not result.success]
 
-        successful_expectations = sum(exp.success for exp in validation_results)
+        successful_expectations = sum(exp.success or False for exp in validation_results)
```

Comment:
why is this required? do some ValidationResults have a `None` value for `success`?